content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Enabling Rate Limits using Envoy
+description: This task shows you how to use Istio to dynamically limit the traffic to a service.
+weight: 10
+keywords: [policies,quotas]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+This task shows you how to use Envoy's native rate limiting in Istio to dynamically limit the traffic to a
+service. In this task, we will apply a global rate-limit for the `productpage` service through ingress gateway that allows
+1 requests per minute across all instances of the service. Additionally, we will apply a local rate-limit for each
+individual `productpage` instance that will allow 10 requests per minute. In this way, we will ensure that our `productpage`
+service handles a maximum of 1 requests per minute through instance gateway, but each `productpage` instance can handle
+up to 10 requests per minute allowing for any in-mesh traffic.
+
+## Before you begin
+
+1. Setup Istio in a Kubernetes cluster by following the instructions in the
+   [Installation Guide](/docs/setup/getting-started/).
+
+1. Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+## Rate limits
+
+Envoy supports two kinds of rate limiting: global and local. Global rate
+limiting uses a global gRPC rate limiting service to provide rate limiting for the entire mesh.
+Local rate limiting is used to limit the rate of requests per service instance.
+Local rate limiting can be used in conjunction with global rate limiting to reduce load on
+the global rate limiting service.
+
+In this task you will configure Envoy to rate limit traffic to a specific path of a service
+using both global and local rate limits.
+
+### Global rate limit
+
+Envoy can be used to [set up global rate limits](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/global_rate_limiting) for your mesh.
+
+1. Global rate limiting in Envoy uses a gRPC API for requesting quota from a rate limiting service.
+   A [reference implementation](https://github.com/envoyproxy/ratelimit) of the API, written in Go with a Redis
+   backend, is used below.
+
+   The following [configmap](https://github.com/envoyproxy/ratelimit#configuration) configures the
+   reference implementation to rate limit requests to the path `/productpage` at 1 req/min and all other
+   requests at 100 req/min.
+
+   {{<text yaml >}}
```

Comment:
This code block must be indented 4 spaces (see lint error)

Reply:
fixed